helm/presets/aws.yaml

```diff
@@ -0,0 +1,44 @@
+receive:
+  service:
+    type: LoadBalancer
+broadcast:
+  service:
+    type: LoadBalancer
+acquire:
+  service:
+    type: LoadBalancer
+strimzi:
+  kafka:
+    resources:
+      requests:
+        cpu: 500m
+        memory: 6Gi
+      limits:
+        cpu: 1000m
+        memory: 8Gi
+  zookeeper:
+    resources:
+      requests:
+        cpu: 100m
+        memory: 512Mi
+      limits:
+        cpu: 250m
+        memory: 1Gi
+redis:
+  enabled: false
+  externalAddress: ""
+presto:
+  postgres:
+    enable: false
+    service:
+      externalAddress: ""
+    db:
+      user: ""
+      password: ""
+global:
+  objectStore:
+    region: "us-east-2"
+    accessKey: ""
```

Comment:
These are set to "false" for me locally

Reply:
`""` is false-y and better suited for what we're trying to do. `false` was me just not knowing better.